.github/workflows/pr-build.yml

```diff
@@ -13,18 +34,33 @@ jobs:
         os:
           - macos-latest
           - ubuntu-latest
-          - windows-latest
+          # Windows is not working for patch workflows, therefore we disable it here
+          # https://github.com/square/wire/issues/2188
+          # https://github.com/open-telemetry/opentelemetry-java/issues/4560
+          # - windows-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-java@v1
+
+      - uses: actions/setup-java@v3
         with:
           java-version: 17
+          distribution: temurin
+
       - uses: gradle/wrapper-validation-action@v1
+
+      - name: Publish patched dependencies to maven local
+        uses: ./.github/actions/patch-dependencies
+        if: ${{ contains(github.event.pull_request.base.ref, 'release/v') }}
```

Comment:
```suggestion
        if: ${{ contains(github.event.pull_request.base.ref, 'release/v*') }}
```

Reply:
No need to add wildcard, but I think instead of contains it is safer to use `startsWith`
https://docs.github.com/en/actions/learn-github-actions/expressions#startswith

Reply:
I will fix the other occurrences where I added the wildcard by mistake :(